### ISSUE TITLE
v1.3.1 release ETA?

### ISSUE THREAD

**Issue by janisozaur** [gulrak/filesystem#59]
OpenRCT2 is a project that uses your library and we've recently received a bug report regarding inability to build on FreeBSD. I saw the support for this platform was added across #49 and #51, but it's only to be included in the upcoming, v1.3.1 release. According to https://github.com/gulrak/filesystem/milestone/17 that seems all done and 15 days overdue already and I wanted to check with you what's the ETA of this release before we start ~~breaking~~applying local changes to the filesystem library?

**Comment by gulrak** [comment]
I had some other work to do, that kept me from making the deadline for v1.3.1, I wanted to work on some internal work for it, but didn't get the time for it. I plan to tag it during this week, I'll postpone  the other changes to v1.3.2.

**Comment by gulrak** [comment]
Not wanting to confuse anyone, but the milestone was wrongly named and I confused myself. The correct name of the release is v1.3.2, as all ghc::filesystem releases have even patch numbers to differentiate them from WIP in between release changes. I renamed the milestone and just pushed out the release v1.3.2.

Sorry for the delay.
